verify/v1/verify.yaml

```diff
@@ -14,52 +14,63 @@ info:
+        step in the Verify process. Without this step Verify process cannot be done.
+        
+        There are two types of Verify criteria templates:
         
-        The following parameter can be configured:
+         - Empty template or,
+         - Customized template
         
-        | Criterion           | Allowed Operations   | Allowed Values                                                                                                           |
-        | ---                 | ---                  | ---                                                                                                                    |
-        | activeNumber        | is                   | true/false                                                                                                               |
-        | numberType          | is, is_in            | fixed or mobile. In case of *is_in* operator multiple comma separated values can be provided                             |
-        | operatorCountry     | is, is_in            | country in ISO Alpha-3 format e.g. DEU, GBR. In case of *is_in* operator multiple comma separated values can be provided |
-        | disposableNumber    | is                   | true/false                                                                                                               |
-        | mccMnc              | is, is_in            | MCC and MNC of the operator e.g. 23202. In case of *is_in* operator multiple comma separated values can be provided      |
+        **Empty Verify criteria template**
+        
+        Empty template means that the default verify and response parameters will be used. This practically means:
+        
+         - Verify criteria
+           - validNumberFormat: Verify that the phone number has a valid format
+           - numberType: Verify that the phone number type is fixed or mobile
+
+         - Response parameters
+           - requestId: Unique request identifier
+           - verified: Verify outcome
+           - reason (only in case of unverified response)
+           - reasonCode (only in case of unverified response)
 
-        From our side, we add "numberType" parameter by default if this parameter is not configured in a Template.
+        **Customized Verify criteria template**
+                
+        Customized templates are more complex to define but more powerful, since customers can define exactly how the
+        Verify service should verify phone numbers and how the response should look like.
         
-        The following parameters can be configured for the response:
+        The following verify parameter can be configured:
         
-        Response Parameter:
+        | Criterion        | Allowed Operations | Allowed Values                                                                                                                  |
```

Comment:
since criteria structure is a bit long and we are showing only one criterion now in creating Template and how to configure that. I think it would be helpful for customers if we provide another column in this table as "Type" to provide information which criteria expect boolean or text. like we have mentioned on this page under Configurable verify criteria: https://confluence.tyntec.com/display/DEV/Verify+Templates.

Just a suggestion :)